modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/EnrichmentManagerSpec.scala

```diff
@@ -697,8 +692,15 @@ class EnrichmentManagerSpec extends Specification with EitherMatchers {
           timestamp,
           RawEvent(api, fatBody, None, source, context)
         )
-        .map(_ => true)
-        .getOrElse(false) must beFalse
+        .swap
+        .forall {
+          case BadRow.EnrichmentFailures(_, failure, _) => failure.messages.map(_.message match {
+            case EnrichmentFailureMessage.Simple(error) => error.startsWith("Enriched event output validation error ")
+            case EnrichmentFailureMessage.IgluError(schemaKey, _) => schemaKey == SchemaKey("com.snowplowanalytics.snowplow", "atomic", "jsonschema", SchemaVer.Full(1, 0, 0))
```

Comment:
Neat: add error message

Reply:
Specs matcher will print the error message. As of feedback round 3.

Reply:
I meant to add a test on the error string contained in the `IgluError`